vesin/include/vesin.h

```diff
@@ -110,11 +114,15 @@ struct VESIN_API VesinNeighborList {
     double (*vectors)[3];
 
     // TODO: custom memory allocators?
+
+    // VesinCUDA specific
+    size_t capacity_cu; /// Capacity of the neighbor list, i.e. how many nnodes can be stored
+    unsigned long * length_cu; /// Pointer to the length of the neighbor list, used for CUDA calculations
```

Comment:
why does this needs to be a pointer?

Reply:
But why does the host needs it? It can be hidden from the public API, allocated on device and synchronized to host once at the end of the call, right?